### PR TITLE
fix(repo): replace remaining instances of yarn with pnpm

### PR DESCRIPTION
**Pull request by meeroslav**
## Related Issue(s)

Fixes #

### CODE REVIEW

File: scripts/commitizen.js [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
-  { value: 'react',         name: 'react:           anything React specific' },
-  { value: 'react-native',  name: 'react-native:    anything React Native specific' },
-  { value: 'expo',          name: 'expo:            anything Expo specific' },
-  { value: 'repo',          name: 'repo:            anything related to managing the repo itself' },
-  { value: 'storybook',     name: 'storybook:       anything Storybook specific' },
-  { value: 'testing',       name: 'testing:         anything testing specific (e.g. jest or cypress)' },
-  { value: 'vite',          name: 'vite:            anything Vite specific' },
-  { value: 'web',           name: 'web:             anything Web specific' },
-  { value: 'webpack',       name: 'webpack:         anything Webpack specific' },
+  { value: 'angular', name: 'angular:         anything Angular specific' },
```
Comment (meeroslav) [inline scripts/commitizen.js:3]: Prettier does this unfortunately

**Review by FrozenPandaz — APPROVED** [review]

File: scripts/commitizen.js (hunk 1 above)
Comment (AgentEnder) [reply scripts/commitizen.js:3]: Jason updated this in a separate PR, can you rebase and undo changes to this file?

Comment (meeroslav) [reply scripts/commitizen.js]: Done, thanks. I need to check why my prettier makes those changes.

**Review by jaysoo — APPROVED** [review]

File: packages/js/src/executors/swc/schema.json [hunk 2]
```diff
@@ -102,7 +102,7 @@
     },
     "generateLockfile": {
       "type": "boolean",
-      "description": "Generate a lockfile (e.g. yarn.lock) that matches the workspace lockfile to ensure package versions match.",
+      "description": "Generate a lockfile (e.g. package-lock.json) that matches the workspace lockfile to ensure package versions match.",
```
[the remark this answers is not in the document]
Comment (meeroslav) [reply packages/js/src/executors/swc/schema.json:105]: The `npm` package manager is far more used, so it makes sense to have that as a default example.
Doesn't break anything, just our docs are full of references to `yarn` despite it not being the most common choice.

**Review by StalkAltan — APPROVED** [review]

**Review by AgentEnder — APPROVED** [review]

**Pull request merged**